### ISSUE TITLE
Uncaught Error: Call to undefined method Dotenv\Repository\RepositoryBuilder::create

### ISSUE THREAD

**Issue by NeilJ247** [Codeception/Codeception#5946]
#### What are you trying to achieve?

Use environment variables as part of setting db module connection details for REST tests.

#### What do you get instead?

Running codeception errors:

```bash
bash-4.2$ ./vendor/bin/codecept -vvv
PHP Fatal error:  Uncaught Error: Call to undefined method Dotenv\Repository\RepositoryBuilder::create() in /var/task/vendor/codeception/codeception/src/Codeception/Lib/ParamsLoader.php:120
Stack trace:
#0 /var/task/vendor/codeception/codeception/src/Codeception/Lib/ParamsLoader.php(44): Codeception\Lib\ParamsLoader->loadDotEnvFile()
#1 /var/task/vendor/codeception/codeception/src/Codeception/Configuration.php(790): Codeception\Lib\ParamsLoader->load()
#2 /var/task/vendor/codeception/codeception/src/Codeception/Configuration.php(181): Codeception\Configuration::prepareParams()
#3 /var/task/vendor/codeception/codeception/src/Codeception/Application.php(65): Codeception\Configuration::config()
#4 /var/task/vendor/codeception/codeception/src/Codeception/Application.php(33): Codeception\Application->readCustomCommandsFromConfig()
#5 /var/task/vendor/codeception/codeception/app.php(39): Codeception\Application->registerCustomCommands()
#6 /var/task/vendor/codeception/codeception/app.php(47): {closure}()
#7 /var/task/vendor/codecept in /var/task/vendor/codeception/codeception/src/Codeception/Lib/ParamsLoader.php on line 120
```

```
params:
  - .env.test
```

```
bash-4.2$ ls .env.test 
.env.test

```

Installed version of vlucas/phpdotenv is v5.1.0.

### Details

* Codeception version: 4.1
* PHP Version: 7.4.4
* Operating System: Linux
* Installation type: Composer

**Comment by NeilJ247** [comment]
Ok I can see the issue.  v5 of  vlucas/phpdotenv is not supported.

```
            if (class_exists('Dotenv\Repository\RepositoryBuilder')) {
                //dotenv v4
                $repository = \Dotenv\Repository\RepositoryBuilder::create()
                    ->withReaders([new \Dotenv\Repository\Adapter\ServerConstAdapter()])
                    ->immutable()
                    ->make();
                $dotEnv = \Dotenv\Dotenv::create($repository, codecept_root_dir(), $this->paramStorage);
            } elseif (method_exists('Dotenv\Dotenv', 'create')) {
                //dotenv v3
                $dotEnv = \Dotenv\Dotenv::create(codecept_root_dir(), $this->paramStorage);
            } else {
                //dotenv v2
                $dotEnv = new \Dotenv\Dotenv(codecept_root_dir(), $this->paramStorage);
            }
```

**Comment by zucler** [comment]
Any updates on this? Laravel 8 is using v5 of `vlucas/phpdotenv` and after upgrading our project, the tests are broken.

**Comment by zucler** [comment]
nvm, just noticed in the changelog that phpdotenv v5 support has been added in Codeception v4.1.8 ([link](https://github.com/Codeception/Codeception/blob/4.1/CHANGELOG-4.1.md))

**Comment by Naktibalda** [comment]
Fixed by #5975
